doc/lib.rst

```diff
@@ -31,9 +31,7 @@ Automatic imports
 
 * `system <system.html>`_
   Basic procs and operators that every program needs. It also provides IO
-  facilities for reading and writing text and binary files. It is imported
-  implicitly by the compiler. Do not import it directly. It relies on compiler
-  magic to work.
+  facilities for reading and writing text and binary files. It is imported implicitly by the compiler. Do not import it directly. It relies on compiler magic to work.
```

Comment:
Did anything actually change here? Anyway, if the original content seems to more-or-less adhere to a certain character limit per line, I think it's best for any modifications to do the same.

Reply:
Yeah @drkameleon I'm not sure if these changes should be there - it's much easier to edit stuff if it's word-wrapped at ~80-90 chars. RST works in such a way that it ignores newlines in these cases.

Reply:
fixed :)